hack/build-antrea-linux-all.sh

```diff
@@ -67,6 +73,10 @@ case $key in
     NO_CACHE=true
     shift
     ;;
+    --build-tag)
+    export BUILD_TAG="$2"
```

Comment:
there should be no need to `export` here, as you are passing `--build-tag $BUILD_TAG"` to the other scripts that are invoked from this one?

Reply:
I have corrected this, actually instead of BUILD_TAG, I need to export the CUSTOM_BUILD_TAG, because that is required for the makefile to tag the images correctly.